content/docs/querying/functions.md

```diff
@@ -80,6 +86,55 @@ and value across all series in the input vector.
 `floor(v instant-vector)` rounds the sample values of all elements in `v` down
 to the nearest integer.
 
+## `histogram_quantile()`
+
+`histogram_quantile(φ float, b instant-vector)` calculates the φ-quantile (0 ≤ φ
```

Comment:
It's the usual symbol for a quantile value. The problem is that usually, quantiles are named the "4-quantile" (for quartiles) and such. specifying the fraction, as we do, is not so common, but if it is done, it's oftes connected to φ. So I think it creates the least amount of confusion.